### ISSUE TITLE
const path = require('path'); Error

### ISSUE THREAD

**Issue by wagg-matt** [imagemin/imagemin#178]
```
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/mattwagg/Sites/unveil-pantheon/unveiluk/wp-content/themes/unveil/gulpfile.js:8:14)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js `(module.js:478:10)
```

Just found this error while installing gulp-imagemin. Any ideas why this is happening?

Node -v
`v0.12.6`

npm -v
`3.9.2`

Currently can't run any gulp script while gulp-imagemin is required.

**Comment by shinnn** [comment]
Upgrade Node.js.

**Comment by wagg-matt** [comment]
Thanks

On 27 May 2016 at 13:24, shinnn <notifications@github.com> wrote:

> Closed #178 <https://github.com/imagemin/imagemin/issues/178>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/imagemin/imagemin/issues/178#event-674389939>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ALE6t9GTG-a-bwr7egZ--qmb964Yb3Fdks5qFuKVgaJpZM4Ioc05>
> .
>



-- 
Matt
